docs/topics/asset/asset.md

```diff
@@ -0,0 +1,55 @@
+---
+description: Asset
+---
+
+# Asset
+
+## Introduction
+
+ASSET is a smart contract token implementation of both EIP-1155 (for limited editions tokens) and EIP-721 (for non fungible, unique tokens)
+
+Each token represents the creations of our players. It is a permission-less implementation of EIP-1155 and EIP-721 where every user can mint their own token represented via metadata.
+
+It implements both EIP-1155 and EIP-721 so player's creation lives in the same id space and can be treated equivalently by wallets or marketplaces that support both EIP. The ID contains all the data of the current token (Creator, is an NFT, number of token when minted, pack ID...).
+
+The ERC1155ERC721 contract share same functionalities as ERC20 and ERC721 (ie get balance, get owner, burn, mint, batch burn, batch mint, approval...).
```

Comment:
`shares`

Reply:
I think we should add a page to explain how the dual ERC1155ERC721 works.
On the sand doc ([check the PR](https://github.com/thesandboxgame/sandbox-smart-contracts/pull/371/files#diff-4f24746e7a4f92fdbfd9c361e2c3348b623edcf31fb87fb0e66bee1f21b0b374)), we decided to add a page under  `docs/topics/token/ERC20.md` to explain our ERC20 implementation, we could do the same for ERC1155ERC721 under `docs/topics/token/ERC1155ERC721.md` and add a link here to that page of course

Reply:
Also can we add links on the gems & catalyst doc to the asset doc when asset is mentionned